padd.sh

```diff
@@ -1387,6 +1387,58 @@ NormalPADD() {
   done
 }
 
+Update() {
+    GetPADDInformation
+
+    if [ "${padd_out_of_date_flag}" = "true" ]; then
+        echo "${full_status_update}"
+
+        padd_script_path="$0"
```

Comment:
I don't think you need those 2 lines
```
        padd_script_file_name="padd.sh"
        padd_script_directory="$(pwd -P)"
```

Reply:
I found them to be required unless I'm missing something here 🤷🏼